routes/album.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import albumController from '../core/controllers/AlbumController.js';
+
+let router = express.Router();
+
+router.get('/:user-id/', albumController.getAlbumsByAuthor);
+router.get('/public/:user-id/', albumController.getAlbumsPublicByAuthor);
+router.get('/like/:album-id/', albumController.getLikesById);
+router.get('/count/:user-id/', albumController.getCountByAuthor);
```

Comment:
What is this route use for?

Reply:
router.get('/:user-id/', photoController.getPhotosByAuthor); Dùng để lấy danh sách album (private và public) của 1 user bất kỳ theo id.
router.get('/public/:user-id/', photoController.getPhotosPublicByAuthor); Dùng để lấy danh sách album (chỉ public) của 1 user bất kỳ theo id.
router.get('/like/:photo-id/', photoController.getLikesById); Dùng để hiển thị số like của 1album(icon trái tym).
router.get('/count/:user-id/', photoController.getCountByAuthor); Dùng để hiển thị tổng số lượng album.